### PR TITLE
Add env file that works with `make brew`

### PR DESCRIPTION
**Pull request by lfdebrux**
To try and make it easier for new devs I've tried to simplify setting up the environment for a dmrunner system on macOS.

I've added an file to source, `Brewfile.env`, that can be used to load the required environment variables if `make brew` has been run. `make brew` will then suggest you source that file and add it to your profile.

---

Example output:

[asciicast](https://asciinema.org/a/vf24YAqvNU4k0MWIlhRjHdO93)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,13 +2,45 @@ SHELL := /bin/bash
...
+	@tput bold
+	@echo -ne '\x33\xEF\xB8\x8F\xE2\x83\xA3  ' # 3
+	@echo -n 'You can now try running `make setup`!'
+	@echo -e ' \x33\xEF\xB8\x8F\xE2\x83\xA3'
+	@tput sgr0
+	@echo
+	@echo 'make setup'
+	@echo
+	@tput bold
+	@echo '😹 Good luck! 😹'
```
Comment (katstevens) [inline Makefile]: Heartily endorse this choice of emoji

**Comment by katstevens** [comment]
I feel like a new dev should try this out? Also, can we add a line to the README about the extra stuff we're magically doing to their machines (adding postgres to the path and so on)?

**Comment by lfdebrux** [comment]
Maybe Joe might be a good guinea pig...

**Comment by lfdebrux** [comment]
Tested with @jonheslop and it seemed to go fairly well 👍

**Pull request merged**